src/main/landing/4 - Transforming/out.yaml

```diff
@@ -0,0 +1,7 @@
+%YAML 1.2
+---
```

Comment:
Not sure if we should add in the example the writer property to not write the declaration.